Makefile

```diff
@@ -1,7 +1,12 @@
 
-build:
+all:
 	./COMPILE-PROTOS.sh
 	go get ./...
 	go install ./...
 	cd extensions/sample; make
 
+test:
+	go test . -v
+	cd plugins; go test . -v
+	cd extensions; go test . -v
+	cd apps/petstore-builder; go test . -v
```

Comment:
This can be simplified with `go test ./... -v`. Unless the goal was to only test specific packages, in which case, `cd` is unnecessary, you can provide the relative path of a package to test:

```
go test . -v
go test ./plugins -v
go test ./extensions -v
go test ./apps/petstore-builder -v
```